librecad/src/main/qc_mdiwindow.cpp

```diff
@@ -449,6 +458,9 @@ bool QC_MDIWindow::slotFileSaveAs(bool &cancelled) {
     QG_FileDialog dlg(this);
     QString fn = dlg.getSaveFile(&t);
 	if (document && !fn.isEmpty()) {
+		QFileInfo info(fn);
+		if (!info.isWritable())
```

Comment:
```isWritable()``` is false for new files, at least on Linux.
This inhibits to save new files.
Try to save the default *Unnamed* drawing, it's impossible.
The app doesn't hang, as you can close with *cancel*, but only without saving modifications.

Reply:
I believe I have fixed both problems you mentioned, but before I submit I'm going to do some more thorough testing, probably with a spreadsheet to make sure I've covered all of the cases.  Hopefully I can begin to redeem myself on the next feature.